Jenkinsfile

```diff
@@ -1,27 +1,31 @@
 // For ci.jenkins.io
 // https://github.com/jenkins-infra/documentation/blob/master/ci.adoc
 
-pipeline {
-    agent {
-        dockerfile {
-            dir 'src/main/resources/ath-container'
-            args '-v /var/run/docker.sock:/var/run/docker.sock'
-            label 'docker && highmem'
-        }
+
+    for (int i = 0; i < (BUILD_NUMBER as int); i++) {
+      milestone()
     }
-    stages {
-        stage('Run ATH') {
-            steps {
-                script { // TODO pending proper solution for JENKINS-43353
-                    for (int i = 0; i < (BUILD_NUMBER as int); i++) {milestone()}
-                }
-                realtimeJUnit(testResults: 'target/surefire-reports/TEST-*.xml', testDataPublishers: [[$class: 'AttachmentPublisher']]) {
-                    sh '''
-                        eval $(./vnc.sh)
-                        ./run.sh firefox latest -Dmaven.test.failure.ignore=true -DforkCount=1 -B
-                    '''
+    def splits = splitTests count(10)
+    def branches = [:]
+    for (int i = 0; i < splits.size(); i++) {
+        int index = i;
+        branches["split${i}"] = {
+            stage("Run ATH - split${index}") {
```

Comment:
For now at least, use a more identifier-like `stage` name, since it makes its way into URLs when reporting test results using newer versions of `junit`. CC @abayer—would be better IMO to use the branch name than a stage label.

Or simply drop `stage` altogether. CC @svanoort & @i386 who were discussing this kind of thing recently.